mlflow/tracking/fluent.py

```diff
@@ -815,6 +816,26 @@ def get_experiment_by_name(name: str) -> Optional[Experiment]:
     return MlflowClient().get_experiment_by_name(name)
 
 
+def list_experiments(
+    view_type: Optional[str] = None, max_results: Optional[int] = None,
+) -> PagedList[Experiment]:
```

Comment:
@harupy @wamartin-aml Can we change the return type to `List[Experiment]`? Callers of the fluent API don't need to reason about pagination. Additionally, it seems that `_paginate` returns a vanilla list type rather than a `PagedList` (see https://github.com/mlflow/mlflow/blob/1c43176cefb5531fbb243975b9c8c5bfb9775e66/mlflow/tracking/fluent.py#L1213), so this return type hint seems incorrect.

Reply:
Thanks for the catch!